### PR TITLE
Lidarr: flac2mp3 release 1.2.1

### PR DESCRIPTION
**Pull request by TheCaptain989**
- Added Variable Bit Rate option
- Fixed cover art conversion issue
- Fixed file permission/owner issue
- Reworked some bash logic
- Updated documentation

### CODE REVIEW

File: .github/workflows/BuildImage.yml [hunk 1]
```diff
@@ -15,8 +15,15 @@ jobs:
 
       - name: Build image
         run: |
-          docker build --no-cache -t ${{ github.sha }} .
-
+          # Build variables
+          VERSION=$(git describe --tags --always)
```
Comment (Roxedus) [inline .github/workflows/BuildImage.yml:19]: This wont return anything useful

Comment (TheCaptain989) [reply .github/workflows/BuildImage.yml:19]: It'll have the first 5 chars of the commit string.
I have this show up in the log on container start, mainly so I can keep track of future bug reports.  But I'm totally open to a better way to tag my builds.

Comment (Roxedus) [reply .github/workflows/BuildImage.yml:19]: I would just use the github.sha, and truncate it, if you dont need the whole string. ie `${GITHUB_SHA:0:5}`

Comment (TheCaptain989) [reply .github/workflows/BuildImage.yml]: Got it.  Updated as requested.
I'll work on a better, easier to maintain human-readable versioning scheme later.

**Review by aptalca — APPROVED** [review]

**Pull request merged**